docs/SECURITY.md

```diff
@@ -0,0 +1,306 @@
+# Security Model of drand
+
+## Table of Contents
+
+* [Notations](#notations)
+* [Model](#model)
+   * [Distributed Key Generation security model](#distributed-key-generation-se
+-model)
+   * [Randomness generation model](#randomness-generation-model)
+* [Attacks](#attacks)
+   * [Randomness Generation](#randomness-generation)
+      * [DDoS the drand network](#ddos-the-drand-network)
+      * [Corruption of the drand network](#corruption-of-the-drand-network)
+   * [Distributed Key Generation](#distributed-key-generation)
+      * [DDoS attacks](#ddos-attacks)
+      * [Corruption attacks](#corruption-attacks)
+      * [Broadcast Channel Assumption](#broadcast-channel-assumption)
+* [TO REVIEW:](#to-review)
```

Comment:
Can you generate this using npmjs.com/doctoc so that it is always up to date? (We use it for many other specs)